draft-ietf-tls-rfc8446bis.md

```diff
@@ -1749,7 +1753,7 @@ appears, it MUST abort the handshake with an "illegal_parameter" alert.
 | max_fragment_length [RFC6066]            |      CH, EE |
 | status_request [RFC6066]                 |  CH, CR, CT |
 | supported_groups [RFC7919]               |      CH, EE |
-| signature_algorithms (RFC8446)           |      CH, CR |
+| signature_algorithms [RFC8446]           |      CH, CR |
```

Comment:
For citations, I generally advise people to use `{{RFC8446}}` rather than `[RFC8446]`.  kramdown-rfc will complain if you have `{{NON-EXISTENT}}`, but just put `[NON-EXISTENT]` straight into the document without a link, so the former is safer.

Reply:
I may fix this later, but for now let's merge.